_returners/splunk_nebula_return.py

```diff
@@ -308,17 +304,17 @@ def flushBatch(self):
 
         if len(self.batchEvents) > 0:
             headers = {'Authorization': 'Splunk ' + self.token}
-            self.server_uri = [x for x in self.server_uri if x[1] is not "bad"]
+            self.server_uri = [x for x in self.server_uri if x[1] is not False]
             for server in self.server_uri:
                 try:
                     r = requests.post(server[0], data=' '.join(self.batchEvents), headers=headers, verify=http_event_collector_SSL_verify, proxies=self.proxy, timeout=self.timeout)
                     r.raise_for_status()
-                    server[1] = "good"
+                    server[1] = True
                     break
                 except requests.exceptions.RequestException:
-                    log.error('Request to splunk server "%s" failed. Marking as bad.' % server[0])
-                    server[1] = "bad"
+                    log.info('Request to splunk server "%s" failed. Marking as bad.' % server[0])
+                    server[1] = False
                 except Exception as e:
-                    log.error('Request to splunk threw an error: {0}'.format(e))
+                    log.info('Request to splunk threw an error: {0}'.format(e))
```

Comment:
An error is something you have to go deal with. An info is something interesting you normally ignore. As the Salt admin, what do I do on a non-AWS host that has errors in this code block? nothing? Then I'd argue info is better.